source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -306,6 +306,77 @@ Before you begin, you'll need a developer sandbox in order to use App Components
 <br>
 That's it! At this point, feel free to keep exploring how changes in the server affects data in the task's Widget. Once you're ready to define an app, [click here to create your own app](/docs/configurations) with App Components.
 
+# Installing an App
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+When a user connects an app with App Components to Asana for the first time, they will go through an installation flow. This involves walking the user through the app's features (i.e., "value props"), authorizing the app, and adding it to projects in Asana. The installation flow can be triggered through either one of two ways:
+
+1. The in-product app gallery. Users can access the app gallery by going into a project (in which they want to install an app), then navigating to **Customize** > **Add App**.
+<br>
+<br>
+<img src="../images/app-gallery.png" alt="app gallery"/>
+<br>
+<br>
+2. The [Asana app directory](https://asana.com/apps):
+<br>
+<br>
+<img src="../images/app-directory.png" alt="app directory"/>
+
+<hr>
+
+### Features
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+After entering the installation flow, the first screen that users see are the app's features, or value props. 
+
+<img src="../images/features.png" alt="features"/>
+
+As part of the customizations, a `valuePropTitle` and `valuePropSubtitle` can be shown at the top of the screen. Additionally, up to three value prop images can be displayed on the screen, each containing the image itself (via an `imageUrl`), accompanying `text` to display under each image, as well as `alt` text.
+
+<hr>
+
+### Authenticating
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+On the next screen, the user will be directed to the auth screen, which will ask them to connect to the external app.
+
+<img src="../images/authentication.png" alt="authentication"/>
+
+When the user clicks the button to continue, they will go through an OAuth flow if the app has configured OAuth. If an [Asana OAuth](/docs/oauth) flow has been configured, it will appear first.
```

Comment:
Do: This is not really correct and we should likely update it. When the user clicks the button to continue, we make a request to the app's specified `authenticationUrl` in a popup window. From there, the developer chooses how to proceed with authentication. Usually this involves going through the Asana OAuth flow as well as a 3rd party OAuth flow (e.g. Jira, Zendesk, Zoom, etc), but technically this is all up to the developer's discretion - they can also include custom screens and logic that prompts the user to fill out additional information for setup of the app. The authentication flow is eventually concluded using the postMessage mechanism that is described elsewhere in the App Components docs.

Reply:
This is makes sense, and we should update the toolkit with this info as well. As far as the technical details of completing the authentication flow, I'll keep the link to https://developers.asana.com/docs/authorization at the end of this section.